src/api/callbackextra.rs

```diff
@@ -0,0 +1,29 @@
+use serde::{Deserialize, Serialize};
+use std::ffi::c_void;
+use steamworks::Callback;
+use steamworks_sys as sys;
+
+const CALLBACK_BASE_ID: i32 = 9000;
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct MicroTxnAuthorizationResponse {
+    /// The reason we were disconnected from the Steam servers
+    pub app_id: u32,
+    pub order_id: u64,
+    // Authorized: 0, Unauthorized: 1
+    pub authorized: u8,
+}
+
+unsafe impl Callback for MicroTxnAuthorizationResponse {
+    const ID: i32 = CALLBACK_BASE_ID + 1;
```

Comment:
Where did you find this ID? It doesn't seems correct looking at the docs

Reply:
thank you for the review, I have checked the code 
https://github.com/Noxime/steamworks-rs/blob/master/steamworks-sys/src/macos_bindings.rs#L3651
should be 152, and I have updated it